apps/test/unit/templates/studioHomepages/homepagesTestData.js

```diff
@@ -167,13 +167,48 @@ export const joinedSections = [
     linkToAssigned: 'https://studio.code.org/s/course4',
     numberOfStudents: 4,
     linkToStudents: manageStudentsUrl,
-    loginType: 'clever',
+    login_type: 'clever',
```

Comment:
why change this back to snake case?

Reply:
I think it was intentional before- these ones are not supposed to render in the unit tests that use them (see screenshot below of the test data codes). I can add a comment in the code indicating as such. 
<img width="330" alt="Screenshot 2023-06-26 at 9 00 04 AM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/533c7b0a-66d1-46c3-831d-4b809fa6368c">

Reply:
Completely agree... I instinctively changed it to match and it failed those unit tests🫠